llvm/test/CodeGen/SPIRV/structurizer/return-early.ll

```diff
@@ -1,6 +1,9 @@
-; RUN: llc -mtriple=spirv-unknown-vulkan-compute -O0 %s -o - | FileCheck %s --match-full-lines
```

Comment:
As an idea, maybe to add `TODO` lines to all those updated tests from `structurizer` where `G_BITCAST` would cause machine verifier to fail? It'd make it easier to switch `-verify-machineinstrs` for them once we figure out `G_BITCAST`.

Reply:
Added TODO where the -verifier was removed (already did in some places, but forgot some).
Not adding it if I didn't removed the flag.
(In the end, we might want to add it anywhere anyway no?)